src/main/java/org/isf/exatype/gui/ExamTypeBrowser.java

```diff
@@ -143,14 +143,11 @@ private JButton getJNewButton() {
 			jNewButton = new JButton();
 			jNewButton.setText(MessageBundle.getMessage("angal.common.new"));
 			jNewButton.setMnemonic(KeyEvent.VK_N);
-			jNewButton.addActionListener(new ActionListener() {
-				
-				public void actionPerformed(ActionEvent event) {
```

Comment:
removing this code means the imports for `ActionEvent` and `ActionListener` are no longer needed